server/portal_server.js

```diff
@@ -101,10 +111,14 @@ app.get('/assets/download/:id', (req, res) => {
             })
             .catch(err => {
               log.info('DeleteError:', id);
+              res.sendStatus(404);
```

Comment:
This will not work. The response has already been sent. Let's talk about what we can do when a delete fails